src/UID2.Client/DecryptionStatus.cs

```diff
@@ -12,5 +12,6 @@ public enum DecryptionStatus
         VersionNotSupported,
         InvalidPayloadType,
         InvalidIdentityScope,
+        UserOptedOut
```

Comment:
i think it's best to call this UserOptedOutDueToClientSideTokenGeneration or UserOptedOutDueToCSTG or something like that - for non-CSTG-derived tokens, the DSPs are supposed to check its internal records and see if the decrypted UID2s have opted out. So here we are really saying User from ONLY CSTG-derived tokens has Opted Out.

Reply:
Worth noting that we plan to use this bit for server opted-out in the coming months, as part of the "policy=1" project

Reply:
@jon8787 so should we leave it as UserOptedOut or make it UserOptedOutDueToCSTG first?

Reply:
I think we should leave it as UserOptedOut but put a bold note in that DSPs are expected to check their records as well because not all opted-out users will have this flag. :(

Reply:
yeh ok leaving it as UserOptedOut then

Reply:
Will leave it as is and add the comment as @jon8787 said